### ISSUE TITLE
urls_to_js output is incorrect when kwargs specified in path()

### ISSUE THREAD

**Issue by bckohan** [bckohan/django-render-static#65]
```
    path(
        'qr/<int:qr>/download',
        QRDownloadView.as_view(),
        kwargs={'format': 'png'},
        name='download_qr'
    ),
    path(
        'qr/<int:qr>/download/png',
        QRDownloadView.as_view(),
        kwargs={'format': 'png'},
        name='download_qr'
    ),
    path(
        'qr/<int:qr>/download/svg',
        QRDownloadView.as_view(),
        kwargs={'format': 'svg'},
        name='download_qr'
    ),
    path(
        'qr',
        QRDownloadView.as_view(),
        kwargs={'format': 'svg'},
        name='download_qr'
    )
```

produces this:

```
"download_qr": (kwargs={}, args=[]) => {
      kwargs={'qr': {'converter': <class 'django.urls.converters.IntConverter'>, 'app_name': 'demoply.qr'}} */
      kwargs={'qr': {'converter': <class 'django.urls.converters.IntConverter'>, 'app_name': 'demoply.qr'}} */
      if (this.match(kwargs, args, ['qr'])) { return `/qr/${kwargs["qr"]}/download/svg`; }
      if (this.match(kwargs, args)) { return "/qr"; }
},
```

**Comment by bckohan** [comment]
Two things broken here:

1) Breadcrumb comment is missing text and open comment /*
2) Reversals via default kwargs should work.
  ```reverse('download_qr', kwargs={'qr': 5, 'format': 'png'}) == 'qr/5/download/png'```

**Comment by bckohan** [comment]
More robust solution will be addressed in #66
